src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Configuration/ConfigUpdaterStep.cs

```diff
@@ -52,7 +54,7 @@ public ConfigUpdaterStep(IEnumerable<IConfigUpdater> configUpdaters, ConfigUpdat
             }
 
             _configFilePaths = (configUpdaterOptions.ConfigFilePaths ?? Enumerable.Empty<string>()).ToArray();
-            SubSteps = configUpdaters.Select(u => new ConfigUpdaterSubStep(this, u, logger)).ToList();
+            SubSteps = _allSteps = configUpdaters.Select(u => new ConfigUpdaterSubStep(this, u, logger)).ToList();
```

Comment:
It seems like `SubSteps` may be better as a virtual property so we could just override it for stuff like this

Reply:
That's a good idea.